content/docs/csidriver/installation/helm/powerflex.md

```diff
@@ -104,7 +104,7 @@ kubectl -n kube-system kustomize deploy/kubernetes/snapshot-controller | kubectl
 ```
 
 *NOTE:*
-- When using Kubernetes 1.21/1.22/1.23 it is recommended to use 5.0.x version of snapshotter/snapshot-controller.
+- When using Kubernetes 1.21/1.22/1.23 it is recommended to use 6.0.x version of snapshotter/snapshot-controller.
```

Comment:
should this be 1.22, 23 and 24 since that is our supported platforms?

Reply:
resolved